person.rb

```diff
@@ -15,10 +15,11 @@ def initialize(
     @id = Random.rand(1...1000)
     @name = name
     @parent_permission = parent_permission
+    @rentals = []
   end
 
```

Comment:
Great job @simplegoose 🤝

- [ ] Kindly ensure a method for adding rentals should be present. `The person itself should be sent as a parameter to create the new rental achieving the 'has-many' association`. You may follow this [link](https://github.com/microverseinc/curriculum-ruby/blob/main/oop/articles/oop_relationships_by_examples.md#many-to-many-relationship) for more clarity.